ykrt/src/profile/linux_perf.rs

```diff
@@ -61,10 +61,13 @@ static JIT_DUMP: LazyLock<Mutex<JitDump>> = LazyLock::new(|| Mutex::new(JitDump:
 /// Create a timestamp of "now" in nanoseconds.
 fn now_timestamp() -> u64 {
     let mut ts = std::mem::MaybeUninit::<timespec>::uninit();
-    // Note: perf docs reccommend using CLOCK_MONOTONIC.
+    // Note: For the JITted code to materialise at the right time in the profile, our clock source
+    // must be the same as perf is using. The perf docs recommend using CLOCK_MONOTONIC, so we use
+    // that. This means that `perf record` must be invoked with `-k CLOCK_MONOTONIC`.
     let ret = unsafe { clock_gettime(CLOCK_MONOTONIC, ts.as_mut_ptr()) };
     if ret != 0 {
-        todo!();
+        // If this fails, we have big problems, and it's probably not worth trying to recover.
```

Comment:
Why not just do:

```
if unsafe { clock_gettime(...) != 0 {
  panic!("Failed to read clock")
}
```

The intermediate variable and the comment don't add much here IMHO.